cmds/test-dev.c

```diff
@@ -0,0 +1,303 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * Test device
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Hubert Badocha
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <hal/hal.h>
+#include <lib/lib.h>
+#include <syspage.h>
+#include <devices/devs.h>
+
+#include "cmd.h"
+
+
+#define TEST_DEV_TIMEOUT_MS 500
+
+#define ERASE_BEFORE 1
+#define ERASE_AFTER  2
+
+#define BUF_SIZE 257 /* Write and read size specially not aligned to any power of 2 to possibly test more edge cases. */
+
+#define CMD_TESTDEV_READ_BIG_BUF_SIZE 0 /* Used to perform a check with a read using big buffer. */
+
+
+static void cmd_testDevInfo(void)
+{
+	lib_consolePuts("performs simple dev read/write test, usage:test-dev [-e erase before] [-E erase after] <dev> <addr> <size>");
+}
+
+
+static void cmd_testDevUsage(void)
+{
+	lib_consolePuts("Usage: test-dev [-e] [-E] <major>.<minor> <addr> <size>\n"
+					"  -e       Erase before\n"
+					"  -E 		Erase after\n");
```

Comment:
Use single tab for line break (excluding `if` case), use spaces for fancy alignment (tabs are for indentation only).
```suggestion
		"  -e       Erase before\n"
		"  -E       Erase after\n");
```

Reply:
Tbh it look very bad without those tabs at the beggining